docs/decisions/0002-events-naming-and-versioning.rst

```diff
@@ -89,3 +90,10 @@ signal until deprecated and removed.
 3. Changing the arguments passed to an event must always be done in a backwards
 compatible way since making it incompatible warrants the use of a new major
 version.
+
+4. Since it was decided from the OEP-50 that Django Signals is the underlying
+mechanism for messages, service developers wanting to listen for events using
+message queues will still require a way to transform the signal-based events
+into the message queue technology. This part of the solution is not covered by
+this library
```

Comment:
NIT: suggested addition `not covered by this library at this time.  In the future we may consider adding it here or setup a separate library for those events.  We leave that decision open for now because many details about the implementation are still unclear.`